docs/src/theme/Footer/index.tsx

```diff
@@ -85,7 +85,9 @@ function FooterSocialLinks(): JSX.Element {
 function FooterPrivacyLinks(): JSX.Element {
 	return (
 		<div className="ffcom-footer-privacy">
-			<LinkItem targetUrl="https://privacy.microsoft.com/privacystatement">Privacy</LinkItem>
+			<LinkItem targetUrl="https://privacy.microsoft.com/privacystatement">
+				Privacy and cookies
```

Comment:
nit : 'Privacy Policy' is the common term used for these kinds of links

Reply:
Saw this requirement in the Privacy review, will confirm with Jade!